LaCE-onboarding-guide.md

```diff
@@ -12,34 +12,85 @@ This page will help you solve common queries that you may face while working on
 ## 1. Creating a Topic 
 - To create a new topic, we first navigate to the `Topics and Skill Dashboard` and select the `CREATE TOPIC` option. To do so, you can take a look at this [guide](Editor-pages.md#steps-to-create-a-topicskill)
 - A modal will open up with necessary details of the topic that are to be filled such as topic name, topic thumbnail, etc.
+
+![topic-modal](https://github.com/SanjaySajuJacob/oppia-web-developer-docs/assets/70763993/038e469a-d236-4147-bfd8-93919e98f587)
```

Comment:
Can you replace the parts in "[...]" with alt text that describes the image, e.g. "Screenshot of the 'New Topic' modal"? Ditto for all other images.

Reply:
Hi @seanlip , I have made the changes as you had mentioned here for all the images. Thanks!